### PR TITLE
EPMDJ-1402. Solved the problem with run admin server on Java>=9.

### PR DESCRIPTION
**Pull request by ViktoriiaLebedeva**

### CODE REVIEW

File: core/src/main/kotlin/com/epam/drill/PluginClassLoader.kt [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package com.epam.drill
+
+import mu.*
+import java.net.*
+
+
+private val logger = KotlinLogging.logger {}
+
+class PluginClassLoader(url:URL) : URLClassLoader(arrayOf(url)) {
```
Comment (IgorKey) [inline core/src/main/kotlin/com/epam/drill/PluginClassLoader.kt]: FORMATTING.

File: core/src/main/kotlin/com/epam/drill/PluginClassLoader.kt [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+    override fun findClass(name: String?): Class<*> {
+        logger.debug {"Loading class with name $name"}
```
Comment (IgorKey) [inline core/src/main/kotlin/com/epam/drill/PluginClassLoader.kt]: so weird. Method 'findClass' but log like loading... why?

File: core/src/main/kotlin/com/epam/drill/PluginClassLoader.kt [hunk 3]
```diff
@@ -0,0 +1,15 @@
+package com.epam.drill
```
Comment (viktorbll) [inline core/src/main/kotlin/com/epam/drill/PluginClassLoader.kt]: The code must reside under the "com.epam.drill.admin" package

File: core/src/main/kotlin/com/epam/drill/admin/PluginClassLoader.kt [hunk 4]
```diff
@@ -0,0 +1,15 @@
+package com.epam.drill.admin
```
Comment (viktorbll) [inline core/src/main/kotlin/com/epam/drill/admin/PluginClassLoader.kt]: com.epam.drill.admin.plugin

**Review by viktorbll — CHANGES_REQUESTED** [review]

File: core/src/main/kotlin/com/epam/drill/plugins/PluginLoaderService.kt [hunk 5]
```diff
@@ -114,11 +115,11 @@ class PluginLoaderService(
...
-        val sysClassLoader = ClassLoader.getSystemClassLoader()
-        sysClassLoader.loadClassesFrom(adminPartFile.toURI().toURL())
+        val pluginClassLoader =
+            PluginClassLoader(adminPartFile.toURI().toURL())
         val entrySet = adminJar.entries().iterator().asSequence().toSet()
         val pluginApiClass =
-            retrieveApiClass(AdminPluginPart::class.java, entrySet, sysClassLoader)
+            retrieveApiClass(AdminPluginPart::class.java, entrySet, pluginClassLoader)
         @Suppress("UNCHECKED_CAST")
         return pluginApiClass as Class<AdminPluginPart<*>>?
```
Comment (viktorbll) [inline core/src/main/kotlin/com/epam/drill/plugins/PluginLoaderService.kt]: `pluginApiClass as? Class<AdminPluginPart<*>>`

File: core/src/main/kotlin/com/epam/drill/plugins/PluginLoaderService.kt [hunk 6]
```diff
@@ -114,11 +115,11 @@ class PluginLoaderService(
     private fun processAdminPart(
         adminPartFile: File, adminJar: JarFile
     ): Class<AdminPluginPart<*>>? {
-        val sysClassLoader = ClassLoader.getSystemClassLoader()
-        sysClassLoader.loadClassesFrom(adminPartFile.toURI().toURL())
+        val pluginClassLoader =
```
Comment (viktorbll) [inline core/src/main/kotlin/com/epam/drill/plugins/PluginLoaderService.kt]: this newline is unnecessary

**Review by viktorbll — APPROVED** [review]

**Pull request merged**